File: test-workspaces/packages/shared_common/taxonomy_aes205_cycle_end.ts
// AES205: Part of circular dependency chain
import { CycleStartEntity } from './taxonomy_aes205_cycle_start';

export class CycleEndEntity {
    data: string = "end";
    process(): string {
        const start = new CycleStartEntity();
        return this.data;
    }
}

File: workspaces-good/modules/shared_common/src/taxonomy_aes205_cycle_end.py
# AES205: Part of circular dependency chain
from ..taxonomy.taxonomy_aes205_cycle_start import CycleStartEntity

class CycleEndEntity:
    def __init__(self):
        self.data = "end"

    def process(self):
        start = CycleStartEntity()
        return self.data
